.github/PULL_REQUEST_TEMPLATE/servicing_pull_request_template.md

```diff
@@ -21,7 +21,3 @@ main PR <!-- Link to PR if any that fixed this in the main branch. -->
 # Risk
 
 <!-- Please assess the risk of taking this fix. Provide details backing up your assessment. -->
-
-# Package authoring signed off?
-
-IMPORTANT: If this change touches code that ships in a NuGet package, please make certain that you have added any necessary [package authoring](../../docs/project/library-servicing.md) and gotten it explicitly reviewed.
```

Comment:
Using myself as the standard for human behavior, I'm concerned with removing this line.

I just opened backports to 8 and 9, wrote the template for 9 and copy/pasted it over 8. Since this line was still at the end of the template I was given I remembered that my library is both in the shared runtime and available as a NuGet package, and did my packaging changes (including the ones that were no longer necessary for 9).

Personally, I recommend leaving a packaging note until 8 goes out of support (which will be after 9 goes out of support), or until 8 also no longer requires manual packaging work.  Perhaps something to the effect of

```
<!-- Packaging changes are no longer required for .NET 9 and beyond.  However, if this change is also being serviced to previous they may be required there. -->
```

Reply:
I'll bring it back but with a different message.

Reply:
Updated.